CHANGELOG-5.0.md

```diff
@@ -7,6 +7,8 @@ in 5.0 minor versions.
 To get the diff for a specific change, go to https://github.com/symfony/symfony/commit/XXX where XXX is the change hash
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v5.0.0...v5.0.1
 
+ * feature add isin constraints in validator component (lmasforne)
+
```

Comment:
Your pull request is related to a new feature (branch `master`), so you have to write this change in [the changelog of the Validator Component (such as line 12)](https://github.com/symfony/symfony/blob/7f53ca55977a463882eded6d0f22b4836d1e3244/src/Symfony/Component/Validator/CHANGELOG.md).
```suggestion
```
This file provides patches on the minor release `5.0` (`5.0.1`, `5.0.2`, ...). When version `5.1` will be released, a new file called `CHANGELOG-5.1.md` will be created.